src/distilabel/progress_bar.py

```diff
@@ -87,12 +89,14 @@ def _generation_progress_func(advance=None) -> None:
         _labelling_progress_func = None
         if has_labeller:
             labelling_progress_bar = get_progress_bar(
-                description="Rows labelled", total=num_rows
+                description="Rows labelled", total=total_rows
             )
 
             def _labelling_progress_func(advance=None) -> None:
                 labelling_progress_bar(advance=1)
 
+            _labelling_progress_func()
```

Comment:
running the examples noticed that the progress_bar never reaches the last element, starts from 0 and doesn't reach the end otherwise. Also unrelated but for a single session in a notebook for example, the progress bar only appears once, that could be tackled in a different PR.